packages/react/src/components/Authenticator/FederatedIdentity/createFederatedIdentity.tsx

```diff
@@ -0,0 +1,16 @@
+import { FederatedIdentityElements } from './context/elements';
+import {
+  UseHandleSignInWithRedirect,
+  FederatedIdentity,
+  CreateFederatedIdentityInput,
+} from './types';
+
+export function createFederatedIdentity<
+  T extends Partial<FederatedIdentityElements>,
+  K extends string = string,
+>(
+  input: CreateFederatedIdentityInput<T, K>
+): {
+  FederatedIdentity: FederatedIdentity<K>;
```

Comment:
As I was looking at `renderIdentity` I realized the name of this component feels kind of confusing, I feel like just from the name I'm not sure what it does.

Reply:
Do you mean the FederatedIdentity component?